lib/WebService/MinFraud/Record/Device.pm

```diff
@@ -26,6 +26,12 @@ has last_seen => (
     predicate => 1,
 );
 
+has local_time => (
+    is        => 'ro',
+    isa       => Str,
```

Comment:
I'm surprised that we use a Str for the time instead of a more restrictive format. It's the same type as `last_seen` though, so seems fine.